packages/create-llama/helpers/python.ts

```diff
@@ -103,22 +103,19 @@ export const installPythonDependencies = (
     );
     const installSuccessful = tryPoetryInstall(noRoot);
     if (!installSuccessful) {
-      console.error(
-        red("Install failed. Please install dependencies manually."),
-      );
-      process.exit(1);
+      const errorMessage =
+        "Poetry installation failed. Please install dependencies manually.";
+      console.log(red(errorMessage));
+      throw new Error(errorMessage);
     }
   } else {
-    console.warn(
-      yellow(
-        `Poetry is not available in the current environment. The Python dependencies will not be installed automatically.
+    const errorMessage = `Poetry is not available in the current environment. The Python dependencies will not be installed automatically.
 Please check ${terminalLink(
-          "Poetry Installation",
-          `https://python-poetry.org/docs/#installation`,
-        )} to install poetry first, then install the dependencies manually.`,
-      ),
-    );
-    process.exit(1);
+      "Poetry Installation",
+      `https://python-poetry.org/docs/#installation`,
+    )} to install poetry first, then install the dependencies manually.`;
+    console.log(red(errorMessage));
```

Comment:
Isn't the thrown error logged again? Wouldn't that result in logging the same message twice?

Reply:
my bad, i added this one with my coroutine code to show the error log during installation and show it again in the final step to help user catch the error easier, but i did remove it so this one is duplicated now. Thanks for the catch!